app/packages/core/src/context/PluginContext.tsx

```diff
@@ -53,6 +56,8 @@ export interface IPluginInstance {
  * all plugin instances, and two methods to get and instance by it's `id` and to get a plugin by it's `type`.
  */
 export interface IPluginContext {
+  getAvailableClusters(): string[];
+  getAvailablePluginTypes(): string[];
```

Comment:
Can we rename these two methods:

```suggestion
  getClusters(): string[];
  getTypes(): string[];
```